### PR TITLE
Create Knack > AGOL signal requests DAG

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/14960

## Associated repo
- https://github.com/cityofaustin/atd-knack-services

## Testing

**Steps to test:**
Start Airflow and trigger `atd_knack_signal_requests`.

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Add note to 1PW secrets moved to API vault and check for duplicates

### CODE REVIEW

File: dags/atd_knack_signal_requests.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+DEFAULT_ARGS = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 1, 1, tz="America/Chicago"),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+    "execution_timeout": duration(minutes=15),
```
Comment (Charlie-Henry) [inline dags/atd_knack_signal_requests.py:21]: I think we're going to bump these all up in the future to 60 minutes minimum to hopefully capture the monthly data refreshes.

File: dags/atd_knack_signal_requests.py
Comment (Charlie-Henry) [inline dags/atd_knack_signal_requests.py:1]: We have this old socrata [dataset](https://data.austintexas.gov/Transportation-and-Mobility/Traffic-and-Pedestrian-Signal-Requests/f6qu-b7zb/about_data), should we somehow note on there that this is archived and no longer being updated?

**Review by Charlie-Henry — APPROVED** [review]

File: dags/atd_knack_signal_requests.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        docker_conn_id="docker_default",
+        api_version="auto",
+        auto_remove=True,
+        command=f"./atd-knack-services/services/records_to_agol.py -a {app_name} -c {container} {date_filter_arg}",
+        environment=env_vars,
+        tty=True,
+        mount_tmp_dir=False,
+    )
+
+    t1 >> t2
```
Comment (Charlie-Henry) [inline dags/atd_knack_signal_requests.py]: git is showing that end of line character missing?

File: dags/atd_knack_signal_requests.py
Comment (johnclary) [reply dags/atd_knack_signal_requests.py:1]: yes—was just chatting with Christina about this morning and she is going to see what AMD wants to do wrt to Socrata. i will add a note—thanks!

Comment (johnclary) [reply dags/atd_knack_signal_requests.py:1]: actually—i'll add the note to the knack-services config.

File: dags/atd_knack_signal_requests.py (hunk 2 above)
Comment (johnclary) [reply dags/atd_knack_signal_requests.py]: thank youu

File: dags/atd_knack_signal_requests.py (hunk 1 above)
Comment (johnclary) [reply dags/atd_knack_signal_requests.py:21]: this one is just 1k records, so i kinda think if it runs for more than 15 minutes it should throw / give up.

File: dags/atd_knack_signal_requests.py
Comment (johnclary) [reply dags/atd_knack_signal_requests.py:1]: ✅  https://github.com/cityofaustin/atd-knack-services/commit/f42faf228b459ac3c64b4aeb0d7e6d5b012bd305

**Review by mddilley — APPROVED** [review]
Looks great - tested locally and I see all the green boxes! 🟩 🚀

**Comment by johnclary** [comment]
this has been deployed on the prod ✅

**Pull request merged**